code/game/objects/items/tools/weldingtool.dm

```diff
@@ -195,7 +195,8 @@
 
 /// Returns the amount of fuel in the welder
 /obj/item/weldingtool/proc/get_fuel()
-	return reagents.get_reagent_amount(/datum/reagent/fuel)
+	var/realFuelNoClickbait = reagents.get_reagent_amount(/datum/reagent/toxin/plasma)
```

Comment:
```suggestion
```

Avoid using vars if you only have a 1 time usage for them. Replace with constant expression